### ISSUE TITLE
Laravel spark install

### ISSUE THREAD

**Issue by YounessTayer** [daem0ndev/laravel-spark#1]
Hi;

I followed your install instructions, except the install folder name, I used "spark" instead of "application".

The "laravel new spark" command was achieved without errors, but in the next step "spark install" command failed.

In fact there is no "spark.php" file nor any file related to Spark application is there.
Do you know what can be the problem?

In advance, thank you;
Youness.

**Comment by phaberest** [comment]
Taylor has closed the Spark source code, so that those instructions will not work anymore.
This is a fork of what it used to be. If you want it, clone the repo.

**Comment by niallobrien** [comment]
So to clarify, is this an open-source fork which will be maintained & updated or is it just for reference/historic purposes? Thanks.

**Comment by phaberest** [comment]
Nope, just a reference you can fork and continue by yourself if you want.

**Comment by niallobrien** [comment]
@phaberest Thanks for clarifying. Perhaps updating the Readme to indicate this would be advisable.
